src/ccontrol/UserQueryProcessList.h

```diff
@@ -111,7 +110,7 @@ class UserQueryProcessList : public UserQuery {
     std::string _getResultOrderBy() const { return _orderBy; }
 
     std::shared_ptr<qmeta::QMetaSelect> _qMetaSelect;
-    qmeta::CzarId const _czarId;  ///< Czar ID in QMeta database
+    CzarId const _czarId;  ///< Czar ID in QMeta database
```

Comment:
The comment "Czar ID in QMeta database" is meaningless (redundant) as the role of the member is completely determined by the name of its type. We have only one concept of czar identity in Qserv.